parcelgen-runtime/src/com/yelp/parcelgen/JsonUtil.java

```diff
@@ -8,99 +10,108 @@
 
-	/**
-	 * If object[key] is a valid integer Unix timestamp,
-	 * returns the appropriate Date for that timestamp.
-	 * Otherwise, returns null.
-	 * @param object
-	 * @param key
-	 * @return
-	 */
-	public static Date parseTimestamp(JSONObject object, String key) {
-		if (object != null && !object.isNull(key)) {
-			try {
-				return new Date(object.getLong(key) * 1000L);
-			} catch (JSONException e) {
-				return null;
-			}
-		} else {
-			return null;
-		}
-	}
+    /**
+     * If object[key] is a valid integer Unix timestamp, returns the appropriate Date for that
+     * timestamp. Otherwise, returns null.
+     */
+    public static Date parseTimestamp(JSONObject object, String key) {
+        if (object != null && !object.isNull(key)) {
+            try {
+                return new Date(object.getLong(key) * 1000L);
+            } catch (JSONException e) {
+                return null;
+            }
+        } else {
+            return null;
+        }
+    }
 
-	/**
-	 * Returns a primitive array of the JSONArray's string values
-	 * or an empty array if the provided array is empty.
-	 * If an entry is invalid or null, the corresponding index
-	 * in the returned array will be <code>null</code>.
-	 * @param array
-	 * @return
-	 */
-	public static String[] getStringArray(JSONArray array) {
-		if (array == null) return new String[0];
+    /**
+     * Returns a primitive array of the JSONArray's string values or an empty array if the provided
+     * array is empty. If an entry is invalid or null, the corresponding index in the returned array
+     * will be <code>null</code>.
+     */
+    public static String[] getStringArray(JSONArray array) {
+        if (array == null) {
+            return new String[0];
+        }
 
-		int len = array.length();
-		String[] list = new String[len];
-		for (int i = 0; i < len; i++) {
-			String obj = array.optString(i, null);
-			if (obj != null) {
-				list[i] = obj;
-			}
-		}
-		return list;
-	}
+        int len = array.length();
+        String[] list = new String[len];
+        for (int i = 0; i < len; i++) {
+            String obj = array.optString(i, null);
+            if (obj != null) {
+                list[i] = obj;
+            }
+        }
+        return list;
+    }
 
-	/**
-	 * Returns a new ArrayList by calling optString() on each
-	 * index in the provided array.
-	 * Empty strings in the input array are preserved.
-	 * @param array
-	 * @return An ArrayList of the elements of array converted to strings,
-	 * or the empty list if array is null.
-	 */
-	public static List<String> getStringList(JSONArray array) {
-		String[] stringArray = getStringArray(array);
-		if (stringArray == null) return Collections.emptyList();
+    /**
+     * Returns a new ArrayList by calling optString() on each index in the provided array. Empty
+     * strings in the input array are preserved.
+     *
+     * @return An ArrayList of the elements of array converted to strings, or the empty list if
```

Comment:
Since you're cleaning up the comments/formatting, we don't need the description and the @return since they say very similar things.